src/test/java/org/codebusters/audiogeek/spotifygazer/domain/spotify/flow/SpotifyNewReleasesFlowAdapterTest.java

```diff
@@ -57,6 +59,9 @@ static void stopWiremock() {
     void getNewReleasesCorrect() throws IOException {
         //given
         var mapper = new ObjectMapper();
```

Comment:
PLease extract ObjectMapper to constant. You do not need to instantiate and configure it for each test method